package-system/squish-ccr/Dockerfile.x86_64

```diff
@@ -0,0 +1,54 @@
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# This docker file uses ubuntu 20.04 as the base image so that the AWS Native C++ libraries will use OpenSSL 3 as the base
+# for its dependencies
+#
+
+FROM amd64/ubuntu:20.04
+ 
+WORKDIR /data/workspace
+
+ARG DOCKER_BUILD_SCRIPT
+
+# Initilize apt cache
+RUN apt-get clean && apt-get update
+
+# Setup time zone and locale data (necessary for SSL and HTTPS packages)
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata locales keyboard-configuration wget gpg
```

Comment:
Why do we need support for the SSL and HTTPS packages in this docker?

Reply:
Its needed for the GPG keys for apt in the block below to get the latest cmake for ubuntu 20.04. The default version for 20.04 had issues with squish-ccr